src/main/resources/modules/CMS130v10.json

```diff
@@ -0,0 +1,281 @@
+{
+  "name": "CMS130",
+  "remarks": [
+    "Synthea module based on CMS130v10 - Colorectal Cancer Screening"
+  ],
+  "states": {
+    "Initial": {
+      "type": "Initial",
+      "direct_transition": "IPP_Age_Guard",
+      "name": "Initial"
+    },
+    "IPP_Age_Guard": {
+      "type": "Guard",
+      "allow": {
+        "condition_type": "And",
+        "conditions": [
+          {
+            "condition_type": "Age",
+            "operator": ">=",
+            "quantity": 50,
+            "unit": "years",
+            "value": 0
```

Comment:
I can't figure out where it came from either but it seems like it can safely be removed. I had re-used these logic conditions from a previously built module that was built using the Synthea Module Builder.